website/docs/development/documentation.md

```diff
@@ -0,0 +1,68 @@
+---
+id: documentation
+title: Documentation
+sidebar_label: Documentation
+---
+
+## Website setup
+
+The website is built using [Docusaurus 2](https://v2.docusaurus.io/).  
+Run the following commands from the `website` directory.
+
+### Install
+
+```
+$ yarn
+```
+### Local Development
+
+```
+$ yarn start
+```
+
+This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+
+For more details, refer [here](https://github.com/facebookresearch/hydra/blob/master/website/README.md).
+
+## NEWS Entries
```

Comment:
Put news entries before websites. More people need to add news entries than to edit the website.